examples/angularjs_require/js/directives/todoEscape.js

```diff
@@ -0,0 +1,16 @@
+/*global define*/
+define(['app'], function (app) {
+	'use strict';
+
+	app.directive('todoEscape', function () {
+		var ESCAPE_KEY = 27;
+
+		return function (scope, elem, attrs) {
+			elem.bind('keydown', function (event) {
+				if (event.keyCode === ESCAPE_KEY) {
```

Comment:
@samccone @passy This is the way other apps [are doing it as well](https://github.com/tastejs/todomvc/blob/master/examples/angularjs/js/directives/todoEscape.js#L18) so maybe we should separate this out in another issue and go forward on this PR?